src/wmtk/TetMesh.h

```diff
@@ -37,10 +37,10 @@ class TetMesh
     // Cell Tuple Navigator
     class Tuple
     {
-        size_t m_global_vid;
-        size_t m_local_eid;
-        size_t m_local_fid;
-        size_t m_global_tid;
+        size_t m_global_vid = -1;
```

Comment:
size_t is an unsigned type... initialize with std::numeric_limits<size_t>::max() instead.